.gitignore

```diff
@@ -1,8 +1,9 @@
+instance_list
```

Comment:
Suggestion:

```
/instance-list*
```

Limits scope to repo root directory only and allows us to append things like a datestamp, uuid, whatever to the filename for preservation of output relative to a specific point in time (see addl. comments below)